src/components/CourtOrderPoa/CourtOrderPoa.vue

```diff
@@ -76,22 +80,26 @@ export default class CourtOrderPoa extends Vue {
     this.$refs.courtNumRef.resetValidation()
   }
 
-  /** Local getter to know when to validate. */
-  private get validateForm (): boolean {
-    return this.validate && this.planOfArrangement
+  /** Triggers the form validation. */
+  public validate (): boolean {
+    return this.$refs.courtNumRef.validate()
   }
 
-  @Watch('validateForm')
+  @Watch('courtOrderNumber')
+  @Watch('planOfArrangement')
+  @Watch('autoValidation')
   validateCourtNum (): void {
-    if (this.validateForm) {
+    if (this.autoValidation) {
       // Apply TextField rules
       this.courtOrderNumRules = [
-        (v: string) => !!v || 'A Court Order number is required',
-        (v: string) => !/^\s/g.test(v) || 'Invalid spaces', // leading spaces
-        (v: string) => !/\s$/g.test(v) || 'Invalid spaces', // trailing spaces
-        (v: string) => !(v.length < 5) || 'Court order number is invalid',
-        (v: string) => !(v.length > 20) || 'Court order number is invalid'
+        (v: string) => (!v || !/^\s/g.test(v)) || 'Invalid spaces', // leading spaces
+        (v: string) => (!v || !/\s$/g.test(v)) || 'Invalid spaces', // trailing spaces
+        (v: string) => (!v || !(v.length < 5)) || 'Court order number is invalid',
+        (v: string) => (!v || !(v.length > 20)) || 'Court order number is invalid'
       ]
+      if (this.planOfArrangement) {
+        this.courtOrderNumRules.splice(0, 0, (v: string) => (!!v && this.planOfArrangement) || 'A Court Order number is required')
```

Comment:
Since courtOrderNumRules is an array, you might find it cleaner to just push another function into it :)

Reply:
Yes. At the beginning I was thinking of the order of the rules but now it doesn't matter anymore. 👍

Reply:
Good point.